Automattic/jetpack
# Add storage access iframe config

Prepare Verbum for 3rd party phaseout

Fixes #
https://github.com/Automattic/jetpack/issues/36559


## Proposed changes:

*

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Sync jetpack and jetpack-mu-wpcom
- Checkout D143612-code
- Enable 3rd party phaseout in `chrome://flags/#test-third-party-cookie-phaseout`
- Open incognito window for: simple site, simple site with custom domain, and atomic site. (jetpack and jetpack-mu-wpcom should be synced on all of them)
- Test the commenting on the opened sites. After allowing storage access in the prompt, the auth request should be sent out and the login state should change in Verbum. Note: on atomic sites this happens when clicking on the form.
